src/PresentationalComponents/RuleDetails/RuleDetails.js

```diff
@@ -93,48 +84,105 @@ const RuleDetails = ({ children, rule, intl, topics, header, isDetailsPage }) =>
                         <hr></hr>
                         <StackItem>{intl.formatMessage(messages.riskofchange)}</StackItem>
-                        <StackItem className='pf-u-display-inline-flex alignCenterOverride pf-u-pb-sm pf-u-pt-sm'>
-                            <span className='pf-u-display-inline-flex'>
-                                <Shield
-                                    hasTooltip={false}
-                                    impact={resolutionRisk}
-                                    size={'md'}
-                                    title={AppConstants.RISK_OF_CHANGE_LABEL[resolutionRisk] || intl.formatMessage(messages.undefined)} />
-                                <span className={`label pf-u-pl-sm ins-sev-clr-${resolutionRisk}`}>
-                                    {AppConstants.RISK_OF_CHANGE_LABEL[resolutionRisk] || intl.formatMessage(messages.undefined)}
-                                </span>
-                            </span>
-                        </StackItem>
-                        <StackItem>
-                            <TextContent>
-                                <Text component={TextVariants.small}>
-                                    {AppConstants.RISK_OF_CHANGE_DESC[resolutionRisk]}
-                                </Text>
-                            </TextContent>
+                        <StackItem className={`pf-u-display-inline-flex alignCenterOverride pf-u-pb-sm pf-u-pt-sm`}>
+                            <Split hasGutter>
+                                <SplitItem>
+                                    <InsightsLabel text={AppConstants.RISK_OF_CHANGE_LABEL[resolutionRisk]} value={resolutionRisk} hideIcon/>
+                                </SplitItem>
+                                <SplitItem isFilled/>
+                                <SplitItem>
+                                    <Stack hasGutter className='description-stack-override'>
+                                        <StackItem>
+                                            <TextContent>
+                                                <Text component={TextVariants.small}>
+                                                    {resolutionRisk ?
+                                                        intl.formatMessage(
+                                                            AppConstants.RISK_OF_CHANGE_DESC[resolutionRisk], {
+                                                                strong(str) { return <strong>{str}</strong>; }
+                                                            }) : intl.formatMessage(messages.undefined)}
+                                                </Text>
+                                            </TextContent>
+                                        </StackItem>
+                                        <StackItem>
+                                            {rule.reboot_required ? <Split>
+                                                <SplitItem>
```

Comment:
split necessary here? think these two can just be placed beside each other in a span? (same for the one below)